airflow/jobs/triggerer_job.py

```diff
@@ -368,8 +518,18 @@ async def run_trigger(self, trigger_id, trigger):
             # allow triggers a chance to cleanup, either in that case or if
             # they exit cleanly.
             trigger.cleanup()
+            self.close_handler(trigger)
 
-    # Main-thread sync API
+            # unsetting ctx_indiv_trigger var restores stdout logging
+            ctx_indiv_trigger.set(None)
```

Comment:
Rather thatn `set(None)` should we use https://docs.python.org/3/library/contextvars.html#contextvars.ContextVar.reset?

Reply:
i explored that. the thing with that is you need to store a token when you set it and then supply the token when resetting.  and i decided it's not worth the complexity, particularly because, if it _were_ set to something other than None, we wouldn't want to set it back to that.